field/TinyLang/Field/Raw/Parser.hs

```diff
@@ -172,24 +172,19 @@ module TinyLang.Field.Raw.Parser
     ( pTop
     ) where
 
-import           TinyLang.Prelude               hiding ( option
-                                                       , many
-                                                       , try
-                                                       )
-import           TinyLang.Field.Raw.Core
+import           TinyLang.Prelude               hiding (many, option, try)
+
+import           Data.Field
 import           TinyLang.Field.Existential
+import           TinyLang.Field.Raw.Core
 import           TinyLang.Field.UniConst
-import           TinyLang.ParseUtils            hiding ( Parser )
-import           Data.Field
+import           TinyLang.ParseUtils
```

Comment:
@jakzale could you commit your `stylish-haskell` config? Otherwise we're going to cancel each other's changes over and over again.

Reply:
@effectfully, any preferences regarding the style?

Reply:
Let's start with what you have right now and if I have any comments, I'll share them here or in slack or whatever.